sysutils/ansible-lint/Portfile

```diff
@@ -4,26 +4,40 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        willthames ansible-lint 3.4.19 v
+github.setup        willthames ansible-lint 5.0.7 v
 name                ansible-lint
 categories          sysutils
 platforms           darwin
 supported_archs     noarch
 license             MIT
-maintainers         artisancomputer.com:zdw \
+maintainers         artisancomputer.com:zdw outlook.de:judaew \
                     openmaintainer
 
 description         Best practices checker for Ansible playbooks
 long_description \
     ansible-lint checks playbooks for practices and behaviour that could potentially be improved
 
-checksums           rmd160  41da918b12b7d3ef9ae7cbee6e07ec653458771b \
-                    sha256  5ef527543ec8a6060da93aa4efcdeb360564242495d0a6e15fcbde071fa5decc
+checksums           rmd160  24ca7e4b1f9417ce9897aabe7d6ed0d9a30f5195 \
+                    sha256  6dc4e592005b7648b47b783fdfcc8f38af6d07c11bc6b13da5316616db198f24 \
+                    size    326613
 
-depends_run-append  path:bin/ansible:py27-ansible
-
-python.versions             27
-python.default_version      27
+python.versions             39
+python.default_version      39
 python.link_binaries_suffix
 
+depends_run-append  path:bin/ansible:py${python.version}-ansible
+depends_lib-append  port:py${python.version}-enrich \
+                    port:py${python.version}-packaging \
+                    port:py${python.version}-ruamel-yaml-clib \
+                    port:py${python.version}-tenacity \
+                    port:py${python.version}-wcmatch \
+                    port:py${python.version}-yaml
```

Comment:
please sort the dependencies alphabetically

Reply:
The dependencies are already sorted alphabetically with `sort`.